recipes/polars-ds/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "polars-ds" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/polars_ds-{{ version }}.tar.gz
+  sha256: 167f29db7b498083fc8c3ac4855bd1bef7711843cd921cb048026bf5d603e729
+
+build:
+  skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+    - make
+  host:
+    - python
+    - maturin >=1.2,<2.0
+    - pip
+
+    # from maturin[patchelf]
+    - patchelf  # [not-win]
```

Do you need this at all? The selector is invalid, so I guess not. If you would need it, it should go into build.
```suggestion
    - patchelf  # [not win]
```

---

I can fix the selector; it might not be needed since it was built fine. Upstream just specified maturin[patchelf] that's why I included it. https://github.com/abstractqqq/polars_ds_extension/blob/v0.3.3/requirements.txt

I can remove it if you think best @xhochy?

---

Better remove it.